src/roles/hostname_check/files/foreman-hostname-check

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+RED=$(tput setaf 1 2>/dev/null)
+GREEN=$(tput setaf 2 2>/dev/null)
+RESET=$(tput sgr0 2>/dev/null)
+
+EXIT_CODE=0
+
+function success() {
+    echo -e "\n${GREEN}[OK]${RESET}\n"
+}
+
+function error() {
+    echo -e "\n${RED}[FAIL]${RESET}\n"
+    CURRENT_EXIT_CODE=$1
+    EXIT_CODE=$((EXIT_CODE|CURRENT_EXIT_CODE))
+    echo -e "$2" >&2
+}
+
+function check-not-localhost() {
```

Comment:
I don't think you can use dashes in function names in bash. If you can, I don't think you should because it's too easy to confuse with subtraction.

Reply:
You can. Bash does not do any substraction by default either.

```
evgeni@juuni:~$ 1-1
bash: 1-1: command not found...
evgeni@juuni:~$ $((1-1))
bash: 0: command not found...
```